### DISCUSSION TITLE
Welcome to Programmers-Overseas-Job-Interview-Handbook Discussions!

### DISCUSSION THREAD

**Discussion by eliaszon** [eliaszon/Programmers-Overseas-Job-Interview-Handbook#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Maidenrate** [comment]
Hi can I join the discussion what's the first question?

**Reply by eliaszon** [reply]
Ofx lol. This discussion forum is pretty new and no one has asked any questions yet.
